src/usr/local/www/interfaces_ppps_edit.php

```diff
@@ -294,26 +294,20 @@
 			}
 		}
 
-		// Loop through fields associated with an individual link/port and make an array of the data
-		$port_fields = array("localip", "gateway", "subnet", "bandwidth", "mtu", "mru", "mrru");
```

Comment:
We do not need to loop through all these fields any more, the only field that has validation is 'mtu'.